.github/workflows/docs-test.yml

```diff
@@ -0,0 +1,27 @@
+name: Check GitHub pages
```

Comment:
What about merging this with `docs.yml`? We can skip the deploy step in PRs with `if: github.event_name == 'push'`.